published/005-2016-Feb-1.md

```diff
@@ -0,0 +1,209 @@
+
+If you would like to get this update as an email, sign up for our [weekly newsletter](//tinyletter.com/ipfsweekly)!
+
+Here are some of the highlights for the February sprints:
+
+## Updates
+
+* [**go-ipfs**](//github.com/ipfs/go-ipfs) 
+  - [@noffle](//github.com/noffle) made [a fix](https://github.com/ipfs/go-ipfs/pull/2296) for when your FUSE connection is terminated.  
+  - [@noffle](//github.com/noffle), [@whyrusleeping](//github.com/whyrusleeping) and [@chriscool](//github.com/chriscool) have been doing lots to make our tests greener. This'll let us get PRs in faster because we won't need to wonder whether our CI is lying to us or not. This gives us more confidence about the code we ship, and allows us to ship more often.
+  - [@noffle](//github.com/noffle) also made progress toward getting `go-ipfs` compatible with vanilla `go get` installation. Hope to have more tangible results to show by end of the month.
+  - [@RichardLitt](//github.com/RichardLitt) made a lot of documentation fixes to the CLI while working on the IPFS HTTP API.
+  - [@lgierth](//github.com/lgierth) improved the automated [docker image](https://github.com/ipfs/go-ipfs/pull/2308) for go-ipfs, which is now named `ipfs/go-ipfs`. Give it a try: `docker run -i --net=host ipfs/go-ipfs`. The new image is built automatically for every commit, and has automatic tags for the coming releases. It's also a few MB smaller, and doesn't require mounting a volume for the IPFS repository anymore. Instead, if there's no volume mounted, it'll generate an ephemeral identity and configuration which will be lost when the container exits. This is ideal for testing or just trying something out real quick. Images for go-ipfs 0.3.x are not provided, because the Dockerfile shipped with these version isn't suitable for Docker Hub automatic builds.
+  - [@chriscool](//github.com/chriscool) rewrote [the build documentation for Windows](//github.com/ipfs/go-ipfs/blob/master/docs/windows.md), which enabled [@slothbag](//github.com/slothbag) to get Windows builds working again with minimal fuss.
+  - [@lgierth](//github.com/lgierth) posted a writeup on the [migration from 0.3.x to 0.4.0](https://ipfs.io/blog/9-v04x-migration/).
+* [**specs**](//github.com/ipfs/specs/pull/37) The IPLD spec was merged, after four months. This reflected a ton of work from [@mildred](//github.com/mildred), [@findkiko](//github.com/findkiko), and [@jbenet](//github.com/jbenet). The [IPLD spec](https://github.com/ipfs/specs/blob/master/merkledag/ipld.md), the "thin-waist" merkle dag format, defines merkle-links, -dags, and -paths, as well as the IPLD Data model and formats. In short: JSON documents with named merkle-links that can be traversed.
```

Comment:
This was mostly work between @mildred and I. @findkiko fixed a minor thing. But many others not mentioned contributed much more in comments and so on. Maybe:

* [**specs**](//github.com/ipfs/specs/pull/37) The IPLD spec was merged, after months of thoughtful design. The bulk of the work was achieved by [@mildred](//github.com/mildred) and [@jbenet](//github.com/jbenet), with lots of comments and design opinions from many other contributors. The [IPLD spec](https://github.com/ipfs/specs/blob/master/merkledag/ipld.md), the "thin-waist" merkle dag format, defines merkle-links, -dags, and -paths, as well as the IPLD Data model and formats. In short: JSON documents with named merkle-links that can be traversed. Stay tuned for more information in the future.